cmd/translate.go

```diff
@@ -0,0 +1,17 @@
+package cmd
```

Comment:
Is it not better to move this file to cmd/translate ?
As far as I understand, now we have command `go run main.go translate` which executes `translate.go`. 

We will have two commands which is `Download` and `Upload`. And upload will be splitted in two parts, upload untranslated text, and translated text. So my idea is something like this:

- cmd
  - root.go
  - translate
    - translate.go (root for translate)
    - download.go (download command)
    - upload.go (upload command)

so if we want to download for example, it will be `go run main.go translate download ...`